controllers/platform/services/services.go

```diff
@@ -271,6 +337,56 @@ func (j JobService) GetServiceCmName() string {
 	return fmt.Sprintf("%s-props", j.GetServiceName())
 }
 
+func (j JobService) SetServiceUrlInStatus(clusterRefPlatform *operatorapi.SonataFlowPlatform) {
+	psJS := NewJobService(clusterRefPlatform)
+	if !isServicesSet(j.platform) && psJS.IsServiceEnabledInSpec() {
```

Comment:
What about the configmap application.properties for the services? shouldn't that be reconciled as well? Removing the data index service will impact the job service application properties.

Reply:
they are! via this change -
https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/add4de1acd82875206bb4de88fb2055122645d99/controllers/sonataflowplatform_controller.go#L225
https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/add4de1acd82875206bb4de88fb2055122645d99/controllers/sonataflowplatform_controller.go#L239-L255

Reply:
> What about the configmap application.properties for the services? shouldn't that be reconciled as well? Removing the data index service will impact the job service application properties.

there will be no impact to a job service configuration as a user only has 2 choices -

1. leverage all cluster-level services
2. leverage none of the cluster-level services

Reply:
ack